iothub/device/src/ITransportSettings.cs

```diff
@@ -17,7 +17,7 @@ public interface ITransportSettings
         TransportType GetTransportType();
 
         /// <summary>
-        /// The default receive timeout.
+        /// The time to wait for a receive operation. The default value is 1 minute.
```

Comment:
I wouldn't add the comment about the default value here. The default isn't relevant for the interface but rather only for the individual implementations.